homeassistant/components/hassio/entity.py

```diff
@@ -58,5 +65,57 @@ def available(self) -> bool:
         """Return True if entity is available."""
         return (
             super().available
+            and DATA_KEY_OS in self.coordinator.data
             and self.entity_description.key in self.coordinator.data[DATA_KEY_OS]
         )
+
+
+class HassioSupervisorEntity(CoordinatorEntity[HassioDataUpdateCoordinator]):
+    """Base Entity for Supervisor."""
+
+    def __init__(
+        self,
+        coordinator: HassioDataUpdateCoordinator,
+        entity_description: EntityDescription,
+    ) -> None:
+        """Initialize base entity."""
+        super().__init__(coordinator)
+        self.entity_description = entity_description
+        self._attr_name = f"Home Assistant Supervisor: {entity_description.name}"
```

Comment:
Using `:` as a separator is pretty uncommon. The only other integration that I'm aware of doing that is `zwave_js`.

Reply:
True, but all existing in this integration have it...

Reply:
Not sure if it's been addressed in the frontend, but having a colon in the name used to stop the frontend from stripping the device name from the entity name in the device page. We'll remove the colon in Z-Wave JS names.

Reply:
![image](https://user-images.githubusercontent.com/15093472/159454095-8feab549-8050-482b-a1c6-8f8fcd0dcfda.png)
Looks fine in the frontend